src/stasis/core.clj

```diff
@@ -200,7 +209,20 @@
           (throw (Exception. (str "URL conflicts between " k1 " and " k2 ": " collisions)))))))
   pages)
 
-(defn merge-page-sources [sources]
+(defn merge-page-sources
+  "Merges collections of pages ensuring every path only occurs once across all
+  collections.
+
+  Takes a map of collection name to page collection, and returns a map of path
+  to content. The collection names are only used for error reporting.
+
+  For example,
+
+      (merge-page-sources
+       {:person-pages (create-person-pages)
+       :article-pages (create-article-pages)
```

Comment:
Whoops! Indentation's off here. I'll fix if this is a change people are interested in.